insights-notification-schemas-java/src/main/java/com/redhat/cloud/notifications/ingress/Parser.java

```diff
@@ -131,18 +139,31 @@ private static void parseFieldIfNeeded(JsonNode container, String field) throws
         }
     }
 
-    private static JsonSchema getJsonSchema() {
+    private static JsonSchema getJsonSchema(boolean relaxed) {
         SchemaValidatorsConfig schemaValidatorsConfig = new SchemaValidatorsConfig();
         schemaValidatorsConfig.setApplyDefaultsStrategy(new ApplyDefaultsStrategy(
                 true,
                 true,
                 true
         ));
 
-        return jsonSchemaFactory().getSchema(
-                Parser.class.getResourceAsStream("/schemas/Action.json"),
-                schemaValidatorsConfig
-        );
+        try {
+            ObjectMapper objectMapper = relaxed ? relaxedObjectMapper : strictObjectMapper;
+
+            JsonNode schema = objectMapper.readTree(Parser.class.getResourceAsStream("/schemas/Action.json"));
```

Comment:
Ideally, the stream opened from the `Action.json` resource should be closed.

Reply:
Done